### ISSUE TITLE
More Quality Profile Customizations

### ISSUE THREAD

**Issue by soulvoid86** [Sonarr/Sonarr#2210]
I would like to see the ability to change the order of Quality in the Quality Profiles, just as we see in Radarr. Please see my screenshot for better example, not sure how to word that.

At the same time, the ability to group Qualities together, so Sonarr isn't double or triple downloading new, but unnecessary releases.

Example:
I download an episode at HDTV720, but a WEBDL720 release is found.  I'm not concerned enough about the very slight quality difference I might get, to waste storage space or download credit on my tracker to worry about that second release. I'd rather wait until a 1080 release is available to redownload that episode. But at the same time, I want to allow all three 720 options because I don't want to miss a release if it's not available in a higher quality just yet.

Basically, add the option to simplify quality down to just 720, 1080, 2k, for those who would like it. The system is great as it is, just would be nice to have both options.

Imgur links to my screenshots:
Radarr: https://imgur.com/oX1PG8j
Sonarr: https://imgur.com/iwkX9uU

Let me know if you need me to word this differently.

Thanks :)

Provide a description of the feature request or bug, the more details the better.
Please use https://forums.sonarr.tv/ for support or other questions. (When in doubt, use the forums)

**Comment by markus101** [comment]
Radarr got that from Sonarr. Turn on advanced settings and you can re-order them.

![image](https://user-images.githubusercontent.com/413544/31241166-ecdf75dc-a9b7-11e7-9a0b-c42762dead21.png)

**Comment by soulvoid86** [comment]
Ahh, I could have sworn I saw that before, but didn't see a way to enable it.
